test/auth/auth_mock_test.go

```diff
@@ -158,13 +158,14 @@ func TestAuthMock_RefreshUserPWAndToken(t *testing.T) {
 			muxToken := http.NewServeMux()
 			muxToken.HandleFunc("/auth", func(w http.ResponseWriter, r *http.Request) {
 				// refresh token cannot be expired
-				assert.True(t, time.Now().Sub(tokenRefreshTime).Seconds() < float64(expirationTimeRefreshToken))
+				log.Printf("%v", time.Since(tokenRefreshTime).Seconds())
+				assert.True(t, time.Since(tokenRefreshTime).Seconds() < float64(expirationTimeRefreshToken))
 
 				tokenRefreshTime = time.Now() // update time when the tokens where refreshed the last time
 				w.Header().Set("Content-Type", "application/json")
 				w.Write([]byte(
 					fmt.Sprintf(`{"access_token": "%v", "expires_in": %v, "refresh_token": "%v", "refresh_expires_in" :  %v}`,
-						AccessToken, expirationTimeToken, RefreshToken, expirationTimeRefreshToken)))
+						AccessToken, uint(12), RefreshToken, expirationTimeRefreshToken)))
```

Comment:
leftover from debugging?

Reply:
This was actually done on purpose to check if the sleeping works correctly

Reply:
could you add a comment explaining why the 12 is there?

Reply:
Done